### ISSUE TITLE
Ripple opacity not working

### ISSUE THREAD

**Issue by Xintium** [stasson/vue-mdc-adapter#343]
I have problems with my ripple effect. It has no transparency anymore, it had it a few updates ago. How would I fix it?

**Comment by pgbross** [comment]
Thanks for submitting your issue, we would need more information to know exactly what you are doing, such as vue-mdc-adapter version, which component you are having trouble with and how you are using it.

It is also possible that the underlying `material-components-web` has changed its behaviour as it is undergoing active development. For example there was a [change to ripple opacity](https://github.com/material-components/material-components-web/pull/2294) in v0.32.0 that could be affecting things.

The best way is to create a `codepen` example such as [this one](https://codepen.io/anon/pen/Brmabr) that demonstrates your issue so we can reproduce the fault.

**Comment by Xintium** [comment]
Hello I use the webpack starter so it is hard to create a codepen.

I use:
material-components-web: 0.33.0
vue-mdc-adapter: 0.13.2

So the problem is probably from material-components-web since the problem was not there when I used an older version. Do you know how I would fix it?

**Comment by pgbross** [comment]
@Xintium 

According to the documentation for [material-components-web ripple](https://github.com/material-components/material-components-web/tree/master/packages/mdc-ripple) you can define your own opacity levels.

In your css define a class that specifies the settings you require and then add that class to the component(s) that you need to customise. 

Their example is:

```
.my-surface {
  @include mdc-ripple-surface;
  @include mdc-ripple-radius-bounded;
  @include mdc-states-base-color(black);
  @include mdc-states-hover-opacity(.1);
  @include mdc-states-focus-opacity(.3);
  @include mdc-states-press-opacity(.4);
}
```

Hope that helps.

**Comment by pgbross** [comment]
@matsp thanks

This looks like the [material-components-web issue](https://github.com/material-components/material-components-web/issues/2458) that describes the problem.

**Comment by Xintium** [comment]
@matsp Thanks that solved the problem

**Comment by pgbross** [comment]
Resolved in v0.14.0
